data/serializers.py

```diff
@@ -98,10 +97,16 @@ class Meta:
         fields = ('id', 'user', 'token', 'endpoint')
 
 
+class ReadOnlyDDSUserCredSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = DDSUserCredential
+        resource_name = 'dds-user-credentials'
+        fields = ('id', 'user', 'endpoint')
```

Comment:
Probably worth a comment here to indicate why this serializer omits key fields (e.g. to suppress our system credentials) and definitely a test case to prove that a standard user does not see these fields through the API.